src/components/Payroll/PayrollOverview/PayrollOverviewPresentation.tsx

```diff
@@ -523,16 +542,56 @@ export const PayrollOverviewPresentation = ({
     },
   ]
 
+  const actions = (
+    <ActionsLayout justifyContent="end">
```

Comment:
Are we sure we want these buttons filling the container on desktop? Seems on desktop we'd want it to have the previous behavior? Maybe worth checking in with design on that?

<img width="720" height="234" alt="Image" src="https://github.com/user-attachments/assets/1aca5761-90f7-4fe6-a48a-86e6a52686f5" />